web/modules/api/generate-api.cjs

```diff
@@ -0,0 +1,59 @@
+const { printer } = require("@spec2ts/core")
+const { generateClient } = require("@spec2ts/openapi-client")
+const fs = require("fs/promises")
+const yaml = require("js-yaml")
+
+async function getSpec(path) {
+  let str = await fs.readFile(path, "utf8")
+  // due to a bug in spec2ts, we're gonna replace `4XX` with `400`
+  str = str.replaceAll(/4XX:/g, "400:")
```

Comment:
Huh, this is strange. What's going on with 4XX errors?

Reply:
Check out https://github.com/touchifyapp/spec2ts/issues/26